web/src/user_group_edit.ts

```diff
@@ -363,22 +410,22 @@ export function handle_member_edit_event(group_id, user_ids) {
     }
 }
 
-export function update_group_details(group) {
+export function update_group_details(group: UserGroup): void {
     const $edit_container = get_edit_container(group);
     $edit_container.find(".group-name").text(group.name);
     $edit_container.find(".group-description").text(group.description);
 }
 
-function update_toggler_for_group_setting() {
+function update_toggler_for_group_setting(): void {
     toggler.goto(select_tab);
 }
 
-export function show_settings_for(group) {
+export function show_settings_for(group: UserGroup): void {
     const html = render_user_group_settings({
         group,
         // We get timestamp in seconds from the API but timerender needs milliseconds.
         date_created_string: timerender.get_localized_date_or_time_for_format(
-            new Date(group.date_created * 1000),
+            new Date(group.date_created ?? 0 * 1000),
```

Comment:
`date_created` field can be `null`. For those we default it to `epoch` (ie. 0)

Reply:
The `??` operator has low [precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#table) and must be parenthesized here.

Reply:
Fixed. Thanks.